src/rt/minfo.d

```diff
@@ -393,9 +393,6 @@ struct ModuleGroup
             ctoridx = 0;
             foreach (int idx, m; _modules)
             {
-                // TODO: Should null ModuleInfo be allowed?
-                if (m is null)
-                    continue;
```

Comment:
I'm pretty sure if ldc passed null modules here, programs would deterministically crash with the current implementation. I don't think we need to worry about it